### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by kirkatronics** [Cardshifter/Cardshifter#452]
The demo site no longer works. Where can I see this game in action?

**Comment by Zomis** [comment]
Yes, unfortunately this project is dead :( I aim to incorporate some of it into my Zomis/Games project.

Contact me on Discord at Zomis#7085 and I can try setting up a temporary server that you can check out.
